tests/test_api.cpp

```diff
@@ -284,6 +284,52 @@ TEST( ApitTest, LoggerApi )
   EXPECT_EQ( receivedLogMessage, "No driver with index: -1" );
 }
 
+TEST( ApiTest, BuildUrisApi )
```

Comment:
this is tested in the driver tests directly, not sure if we need it here at all. in the test_api.cpp you should test more some iff cases like invalid filename, empty filename, non-existent driver name and so on...

Reply:
I added more edge cases